packages/opencensus-core/src/metrics/export/types.ts

```diff
@@ -266,15 +266,16 @@ export interface SummaryValue {
    * The total number of recorded values since start_time. Optional since
    * some systems don't expose this.
    */
-  readonly count: number;
+  readonly count?: number;
```

Comment:
Remove optional (?) from these properties.

Reply:
> Remove optional (?) from these properties.

   * The total number of recorded values since start_time. Optional since
   * some systems don't expose this.

Reply:
Done